swiggy-pdp-cart/src/components/Main/Content/Content.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+import Menu from "./Menu/Menu";
+import Cart from "./Cart/Cart";
+import Categories from "./Categories/Categories";
+import { fetchCategoriesList,fetchMenuList } from "../../../services/fakeFetchFunctions";
+import { categoryNameById } from "./helper/helper";
+import ContentCSS from './content.module.css'
+export default function Content(){
+  
+  const [category,setCategory]=useState('recommended');
+  const [menu,setMenu]=useState([]);
+  const [categoryList,setCategoryList]=useState([]);
+  
+  useEffect(() => {
+    const menu = fetchMenuList();
```

Comment:
While this is not wrong, this will not help you understand how to make the API calls correctly.

Either integrate the API call from node server, or make your fetch functions return a promise so that you can mimic the async flow here.

Once done, handle it appropriately here.